API/requirements.txt

```diff
@@ -2,4 +2,6 @@
 # https://pip.pypa.io/en/stable/reference/requirements-file-format/
 pycodestyle==2.8.0
 autopep8==1.6.0
-Flask==2.0.3
\ No newline at end of file
+Flask==2.0.3
+peewee=3.14.10
```

Comment:
A small typo but we can proceed, I have another draft pull request ready to fix it